src/bentoml/_internal/frameworks/keras.py

```diff
@@ -269,7 +269,12 @@ def KerasSequentialModel() -> keras.models.Model:
         metadata=metadata,
         signatures=signatures,
     ) as bento_model:
-        model.save(bento_model.path, include_optimizer=include_optimizer, **kwargs)
+        model.save(
```

Comment:
Actually one more nit check: should have two branch to check if keras > 3.4, then use yours args, otherwise keep the previous behaviour.

Reply:
Done!